backend/FwHeadless/FwHeadlessConfig.cs

```diff
@@ -15,6 +15,7 @@ public class FwHeadlessConfig
     public required string LexboxPassword { get; init; }
     [Required]
     public required string ProjectStorageRoot { get; init; }
+    public string MaxUploadFileSizeKb { get; init; } = "10240";
```

Comment:
why not make this an int?

Reply:
Because as I understood the docs, they said that values in an `IOptions<FooConfig>` must be strings, and any parsing would be done by your app later on. Was I misreading it?

Reply:
The docs I was relying on are https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-9.0#configuration-keys-and-values where it says:

> Configuration values:
> * Are strings.
> * Null values can't be stored in configuration or bound to objects.

No mention of allowing config values that are ints or other data types.

Reply:
that's talking about how they are stored in IConfiguration, not when they are bound to an object. For example `HealthChecksConfig.RequireFwHeadlessContainerVersionMatch` is a bool and `SmtpPort` in email config is an int. Configuration can be thought about from where things are set. However [options](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-9.0) are how you consume configuration in a type safe way. That's why they don't reference what types can be bound on the config page.

Reply:
Done in commit ca5a51ec99.